util/stream.go

```diff
@@ -0,0 +1,19 @@
+package util
+
+import "crypto/cipher"
+
+// ConstantStream is a cipher.Stream which always returns
+// the same value.
+type ConstantStream struct {
+	seed []byte
+}
+
+// NewConstantStream returns a ConstantStream seeded with buff
+func NewConstantStream(buff []byte) cipher.Stream {
```

Comment:
Just rename this function to ConstantStream() and make it return a value of a hidden type, since there's no reason for the caller to be aware of anything but the cipher.Stream interface to it.